packages/openapi-v3/src/controller-spec.ts

```diff
@@ -80,7 +81,7 @@ function resolveControllerSpec(constructor: Function): ControllerSpec {
 
     const endpoint = endpoints[op];
     const verb = endpoint.verb!;
-    const path = endpoint.path!;
+    const path = toOpenApiPath(endpoint.path!);
```

Comment:
Hmm ... why is `path` not a required property? (If it was we wouldn't need the `!`.

Reply:
If `path` is missing, it's default to `/`'.